pages/science/reactions.rst

```diff
@@ -285,6 +285,47 @@ Chebyshev reactions can be defined in the CTI format using the
 :cti:class:`chebyshev_reaction` entry, or in the YAML format using the
 :ref:`Chebyshev <sec-yaml-Chebyshev>` reaction ``type``.
 
+.. _sec-Blowers-Masel:
+
+Blowers-Masel Reactions
+-----------------------
+
+Blowers-Masel rate expression represents the approximation porposed by Blowers
+and Masel [#BlowersMasel2000]_ to automatically scale activation energy
+as we change species' enthalpies. The activation energy estimation can be written
```

Comment:
How are species' enthalpies changed? Can you give a one-sentence description of how or why that might happen?